docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -190,33 +222,44 @@ You can read more about decay functions
 NOTE: Decay functions on dates are limited to dates in the default format
 and default time zone. Also calculations with `now` are not supported.
 
-===== Functions for vector fields
+[[script-score-functions-vector-fields]]
+====== Functions for vector fields
 <<vector-functions, Functions for vector fields>> are accessible through
 `script_score` query.
 
-==== Faster alternatives
-Script Score Query calculates the score for every hit (matching document).
-There are faster alternative query types that can efficiently skip
-non-competitive hits:
+[[script-score-faster-alt]]
+===== Faster alternatives
+The `script_score` query calculates the <<relevance-scores,relevance score>> for
+every matching document, or hit. There are faster alternative query types that
+can efficiently skip non-competitive hits:
 
-* If you want to boost documents on some static fields, use
- <<query-dsl-rank-feature-query, Rank Feature Query>>.
+* If you want to boost documents on some static fields, use the 
+ <<query-dsl-rank-feature-query, `rank_feature`>> query.
+ * If you want to boost documents closer to a date or geographic point, use the
+ <<query-dsl-distance-feature-query, `distance_feature`>> query.
 
+[[script-score-function-score-transition]]
+===== Transition from the function score query
+We are deprecating the <<query-dsl-function-score-query, `function_score`>>
+query. We recommend using the `script_score` query instead.
 
-==== Transition from Function Score Query
-We are deprecating <<query-dsl-function-score-query, Function Score>>, and
-Script Score Query will be a substitute for it.
+You can implement the following functions from the `function_score` query using
+the `script_score` query:
 
-Here we describe how Function Score Query's functions can be
-equivalently implemented in Script Score Query:
+* <<script-score>>
+* <<weight>>
+* <<random-score>>
+* <<field-value-factor>>
+* <<decay-functions>>
+* <<vector-functions>>
```

Comment:
<<vector-functions>> were not a part of the original `function_score` query, so we can exclude it from this list.

Reply:
Fixed with c527ec1612ed47435fe6b6180d3443cfedffe3ad.